streaming/vision/__init__.py

```diff
@@ -1,5 +1,8 @@
 # Copyright 2022 MosaicML Streaming authors
 # SPDX-License-Identifier: Apache-2.0
 
-from .cifar10 import CIFAR10 as CIFAR10
-from .imagenet import ImageNet as ImageNet
+from streaming.vision.ade20k import ADE20k as ADE20K
+from streaming.vision.cifar10 import CIFAR10 as CIFAR10
+from streaming.vision.imagenet import ImageNet as ImageNet
+
+__all__ = ['ADE20K', 'CIFAR10', 'ImageNet']
```

Comment:
TODO: add __all__ to streaming/text/__init__.py

Reply:
Done in https://github.com/mosaicml/streaming/pull/9/commits/6972e99881cba28a7035b300973d58e07a5b0509